lldb/source/Plugins/SymbolFile/DWARF/DWARFBaseDIE.h

```diff
@@ -46,6 +48,7 @@ class DWARFBaseDIE {
   explicit operator bool() const { return IsValid(); }
 
   bool IsValid() const { return m_cu && m_die; }
+  bool isValid() const { return IsValid(); }
```

Comment:
How about using `operator bool` as the common api ?

Reply:
I'm sorry, I guess I wasn't clear about this. What I was suggesting is to just revert the addition `isValid` -- as there already exists a comon API. I didn't want to remove things that are already there. (I mean, I'd be fine with that doing that under the flag of moving the LLDB API closer to LLVM, but I don't think we ought to do that as a part of this PR)